### PR TITLE
feat(Logo): update component, story and screenshot tests

### PR DESCRIPTION
**Pull request by marcoskolodny**
- Adding data attributes to component
- Fixing svg container's width
- Updating story and screenshot tests

### CODE REVIEW

File: src/__stories__/logo-story.tsx
Comment (yceballost) [inline src/__stories__/logo-story.tsx:1]: I liked the part of the old story where you could see that you could force the logos with `<MovistarLogo />`, `<VivoLogo />`, etc

How can we keep it?

Comment (marcoskolodny) [reply src/__stories__/logo-story.tsx:1]: That's actually possible with the new implementation. It's the control named "skin"

Comment (marcoskolodny) [reply src/__stories__/logo-story.tsx:1]: Discussed offline. Updated controls in order to use a boolean `forceBrandLogo` that allows you choose and show a specific brand.

File: src/__stories__/logo-story.tsx [hunk 1]
```diff
@@ -1,82 +1,98 @@
...
 };
 
-type Args = {type: 'isotype' | 'imagotype' | 'vertical'; size: number; inverse: boolean};
+type Args = {
+    type: 'isotype' | 'imagotype' | 'vertical';
+    size: number;
+    inverse: boolean;
+    action: string;
+    forceBrandLogo: boolean;
+    brand: 'Movistar' | 'O2' | 'Vivo' | 'Vivo-new' | 'Telefonica' | 'Blau';
```
Comment (yceballost) [inline src/__stories__/logo-story.tsx:19]: you can remove `Vivo-new` from this selector, the brand for Vivo and Vivo-new skin is Vivo :)

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/__stories__/logo-story.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/__stories__/logo-story.tsx]: Good catch. Fixed

**Review by yceballost — APPROVED** [review]

File: src/__stories__/logo-story.tsx [hunk 2]
```diff
@@ -1,82 +1,97 @@
...
     },
-    inverse: false,
 };
 
-type Args = {type: 'isotype' | 'imagotype' | 'vertical'; size: number; inverse: boolean};
+type Args = {
+    type: 'isotype' | 'imagotype' | 'vertical';
+    size: number;
+    inverse: boolean;
+    action: string;
```
Comment (atabel) [inline src/__stories__/logo-story.tsx:17]:
```suggestion
    action: 'none' | 'onPress' | 'href' | 'to';
```

**Review by atabel — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**